src/util/BaseLoginRouter.js

```diff
@@ -199,9 +205,10 @@ function (Okta, Backbone, xdomain, RefreshAuthStateController, Settings, Header,
 
         if (!oldController) {
           this.el.append(this.controller.el);
+          this.controller.triggerEvent();
```

Comment:
Also, I think we should call this function something more generic (again, in line with the idea that this is just the first of many events we are going to be publishing).

How about:
```javascript
this.controller.postAnimation();
```